docs/src/create-apps/upgrading.md

```diff
@@ -278,7 +278,7 @@ the configuration files format. Most of the old configuration format is
 still supported, but customers are invited to move to the new format.
 
 For an example upgrade path, see the [Drupal 7.x branch of the
-Platformsh-examples
+`platformsh-examples`
```

Comment:
This spelling triggered a Vale failure. Let me know if you'd like to include as is, place an exception somewhere else, or address in some other way.